components/exhibition/ExhibitionChaptersNavigation.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="my-3 d-flex justify-content-end">
+    <b-link
+      v-if="previousChapter"
+      :to="previousChapter.url"
+      class="chapter-nav prev"
+      data-qa="previous chapter button"
+    >
+      {{ previousChapter.name }}
+    </b-link>
+    <b-link
+      v-if="nextChapter"
+      :to="nextChapter.url"
+      class="chapter-nav next"
+      data-qa="next chapter button"
+    >
+      {{ nextChapter.name }}
+    </b-link>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: {
+      exhibitionIdentifier: {
+        type: String,
+        required: true
+      },
+      chapterNavigation: {
+        type: Array,
+        required: true
+      }
+    },
+    data() {
+      return {
+        currentChapter: this.$route.params.chapter
+      };
+    },
+    computed: {
+      currentChapterIndex() {
+        return this.chapterNavigation.findIndex(chapter => chapter.identifier === this.currentChapter);
+      },
+      nextChapter() {
+        const nextIndex = this.currentChapterIndex + 1;
+        return this.getChapter(nextIndex);
+      },
+      previousChapter() {
+        const prevIndex = this.currentChapterIndex - 1;
+        return this.getChapter(prevIndex);
+      }
+    },
+    methods: {
+      getChapter(index) {
+        return this.chapterNavigation[index];
```

Comment:
This method now seems like overkill given its simplicity. Just have the computed methods return the relevant chapter themselves.